Formula/mlpack.rb

```diff
@@ -0,0 +1,63 @@
+class Mlpack < Formula
+  desc "Scalable C++ machine learning library"
+  homepage "https://www.mlpack.org"
+  url "https://mlpack.org/files/mlpack-3.2.2.tar.gz"
+  sha256 "7aef8c27645c9358262fec9ebba380720a086789d6519d5d1034346412a52ad6"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+  depends_on "armadillo"
+  depends_on "boost"
+  depends_on "ensmallen"
+  depends_on "graphviz"
+
+  def install
+    cmake_args = std_cmake_args
+    cmake_args << "-DDEBUG=OFF"
+    cmake_args << "-DPROFILE=OFF"
+    cmake_args << "-DDOWNLOAD_STB_IMAGE=OFF"
```

Comment:
Do you think we could create a package for stb as well? Would be nice if a user could load/save images.

Reply:
It looks liek this has been a discussion on the stb repo before -- https://github.com/nothings/stb/issues/359 I don't think it will work with brew as is due to a lack of a tarball that I could identify on their repo. However, if you know if a tarball we could use for stb, then let's do it.

Reply:
I've added https://github.com/Homebrew/homebrew-core/pull/48121; if that's approved, I'll use it here.

Reply:
I took a quick glance at the `stb` pull request and I'm disinclined to include it without a tagged release. Reading nothings/stb#359 it seems the better solution would be to include the necessary header files as a `resource` and then point cmake in the right direction.

Reply:
Using `resource` sounds like a good idea to me.

Reply:
I've added this in my last force-commit. So, as soon as I can stop messing up git rebase, it should be good ;)

Reply:
Agreed, I've had success just packaging `stb_image.h` and `stb_image_write.h` as part of the mlpack itself for other package managers (i.e. Fedora/RHEL).